src/main/java/animals/birds/Owl.java

```diff
@@ -1,21 +1,19 @@
 package animals.birds;
 
-import animals.Animals;
-
-public abstract class Birds extends Animals {
-    public Birds(String animalName, String typeOfFood) {
+public class Owl extends Bird {
+    public Owl(String animalName, String typeOfFood) {
         super(animalName, typeOfFood);
     }
-
+    @Override
+    public String feature() {
+        return "спит днем";
+    }
     @Override
     public String moveMethod() {
```

Comment:
The moveMethod and breatheMethod overrides belong to the Bird class if you ask me. Note how you have to override them all the same for every bird type, instead you can add these overrides to the Bird class and all the birds will inherit it.

Same for other creatures